frontend/views/containers/sidebar/GroupMembers.vue

```diff
@@ -89,6 +115,10 @@ export default {
   align-items: end;
 }
 
+.c-group-list {
+  margin-bottom: $spacer-md + $spacer-sm;
```

Comment:
that's confusing. Better use direct value I reckon or leave a comment on the actual size

Reply:
Based on the designs, it was neither `md` nor `sm`, it was the sum of both.

Reply:
so that's why I did that sum (it will be compiled to the final value anyway), instead of putting a hardcoded value